tests/informatics/test_predictors.py

```diff
@@ -110,3 +122,30 @@ def test_expression_post_build(expression_predictor):
     assert session.get_resource.call_count == 1
     assert expression_predictor.report is not None
     assert expression_predictor.report.status == 'OK'
+
+
+def test_molecule_featurizer(molecule_featurizer):
+    assert molecule_featurizer.name == "Molecule featurizer"
+    assert molecule_featurizer.description == "description"
+    assert molecule_featurizer.descriptor == MolecularStructureDescriptor("SMILES")
+    assert molecule_featurizer.features == ["standard"]
+    assert molecule_featurizer.excludes == []
+
+    assert str(molecule_featurizer) == "<MolecularStructureFeaturizer 'Molecule featurizer'>"
+
+    # This is what you get when you require 100% test coverage
+    assert molecule_featurizer._post_dump(molecule_featurizer.dump())["display_name"] == molecule_featurizer.name
```

Comment:
Why is this needed for coverage? Shouldn'T `molecule_featurizer.dump()` automatically call the code in `_post_dump()`

Reply:
🤷‍♂️without this, test coverage for _post_dump wasn't detected. I'll check to see if dump() is covered.

Reply:
dump() wasn't covered - that did it 👍